core/renderers/zelos/info.js

```diff
@@ -202,6 +215,44 @@ Blockly.zelos.RenderInfo.prototype.addAlignmentPadding_ = function(row,
   }
 };
 
+/**
+ * Adjust the x position of fields to bump all non-label fields in the first row
+ * past the notch position.
+ * @protected
+ */
+Blockly.zelos.RenderInfo.prototype.adjustXPosition_ = function() {
+  if (!this.topRow.hasPreviousConnection) {
+    return;
+  }
+  var firstInputRow = null;
+  for (var i = 0, row; (row = this.rows[i]); i++) {
+    if (!firstInputRow && Blockly.blockRendering.Types.isInputRow(row)) {
+      firstInputRow = row;
```

Comment:
Instead of tracking `firstInputRow` you can just return after handling the relevant row.

Reply:
Yep, must've been an artifact of refactoring from ``finalize_``.